src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+    let count = 0;
+
+    for (const mass of this) {
```

Comment:
The `mass` name is not meaningful enough for the variable. Use more meaningful names, please